NickvisionMoney.Shared/Models/AccountMetadata.cs

```diff
@@ -121,6 +124,24 @@ internal AccountMetadata(string name, AccountType accountType)
         SortTransactionsBy = SortBy.Date;
     }
 
+    /// <summary>
+    /// The style to use for displaying an amount
+    /// </summary>
+    /// <remarks>Must be a value between 0 and 4. See https://learn.microsoft.com/en-us/dotnet/api/system.globalization.numberformatinfo.currencypositivepattern?view=net-7.0#remarks for the values' meaning</remarks>
+    public int? CustomCurrencyAmountStyle
+    {
+        get => _customCurrencyAmountStyle;
+
+        set
+        {
+            if (value != null && (value < 0 || value > 4))
```

Comment:
`value > 3`?

Reply:
Ah yes... could you change that and the comment for me?